examples/misc/histogram.html

```diff
@@ -0,0 +1,117 @@
+  <!-- Include Mapbox GL JS -->
+  <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+  <!-- Include Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="../style.css">
+</head>
+
+<body>
+  <!-- Map goes here -->
+  <div id="map"></div>
+  <aside class="chart-toolbox">
+    <div class="box">
+      <header>
+        <h1>Histogram</h1>
+      </header>
+      <section>
+        <div id="controls">
+          <canvas id="histogram" width="500" height="600"></canvas>
+        </div>
+      </section>
+      <footer class="js-footer"></footer>
+    </div>
+  </aside>
+  <div id="loader">
+    <div class="CDB-LoaderIcon CDB-LoaderIcon--big">
+      <svg class="CDB-LoaderIcon-spinner" viewBox="0 0 50 50">
+        <circle class="CDB-LoaderIcon-path" cx="25" cy="25" r="20" fill="none"></circle>
+      </svg>
+    </div>
+  </div>
+
+  <script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+      center: [2.212774913887074, 41.416951606921714],
+      zoom: 11.5,
+      dragRotate: false,
+    });
+    map.touchZoomRotate.disableRotation();
+
+    carto.setDefaultAuth({
+      user: 'cartovl',
+      apiKey: 'default_public'
+    });
+
+    const ctx = document.getElementById('histogram').getContext('2d');
+
+    // Define the source
+    const source = new carto.source.Dataset('spend_data');
+
+    // Define the visualization
+    const viz = new carto.Viz(
+      `
+      @histogram: viewportHistogram($category)
+      color: ramp($category, bold)
+      width: 5
+      strokeWidth: 0
+    `
+    );
+
+    const layer = new carto.Layer('layer', source, viz);
+
+    layer.addTo(map, 'watername_ocean');
+
+    layer.on('loaded', () => {
+      hideLoader();
+    });
+
+    layer.on('updated', () => {
+      const histogram = layer.viz.variables.histogram.eval();
+      const colorRamp = layer.viz.color;
+      const labels = histogram.map(value => value.x);
+      const data = histogram.map(value => value.y);
+      const backgroundColor = colorRamp
+        .getLegend()
+        .data
+        .map(legend => computeColor(legend.value));
+
+      var myChart = new Chart(ctx, {
+        type: 'horizontalBar',
+        options: {
+          legend: {
+            display: false
+          }
+        },
+        data: {
+          labels,
+          datasets: [{
+            data,
+            backgroundColor
+          }]
+        }
+      });
+    });
+
+    function hideLoader() {
+      document.getElementById('loader').style.opacity = '0';
+    }
+
+    function computeColor(color) {
+      return `rgba(${color.r},${color.g},${color.b},${color.a})`;
+    }
+
+  </script>
+</body>
+
+</html>
```

Comment:
✨ The example looks pretty nice!

Bar animations are smooth when changing zoom, though sometimes they present a little "trembling" when using drag. Maybe we should add a **debounce**